src/representations/representation_manager_spherical_expansion.hh

```diff
@@ -773,7 +773,6 @@ namespace rascal {
     SparsePropertyGradient_t expansions_coefficients_gradient;
 
    protected:
-   private:
```

Comment:
Uh-oh... who's inheriting from RepresentationManagerSphericalExpansion that should have access to all the internal variables?